### PR TITLE
sysutils/chericat: initial port of chericat

### PR DESCRIPTION
**Pull request by brooksdavis**
Port skeleton for chericat. It builds and installs the main binary. It seems likely we want to install various python and jupyter bits as well, but I'd need some guidance on what makes sense to install where.

### CODE REVIEW

**Comment by brooksdavis** [comment]
This is probably blocked on https://github.com/CTSRD-CHERI/chericat/issues/6

**Comment by InterestedInTechAndCake** [comment]
PR https://github.com/CTSRD-CHERI/cheribsd-ports/pull/120 should be merged into this PR before the changes here can be merged to main.

File: sysutils/chericat/Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+COMMENT=	Capability Analysis Tool for CHERI
+WWW=		https://github.com/CTSRD-CHERI/chericat
+
+LICENSE=	BSD2CLAUSE
+
+USE_GITHUB=	yes
+GH_PROJECT=	chericat
+GH_ACCOUNT=	CTSRD-CHERI
+GH_TAGNAME=	${CHERICAT_COMMIT}
+
```
Comment (kwitaszczyk) [inline sysutils/chericat/Makefile:15]:
```suggestion

LIB_DEPENDS=	libsqlite3.so:databases/sqlite3

```

**Comment by kwitaszczyk** [comment]
The first commit has the author set to "brooks davis <brooks@cheribsd.localdomain>" which probably should be fixed.

**Comment by InterestedInTechAndCake** [comment]
PR https://github.com/CTSRD-CHERI/cheribsd-ports/pull/121 is needed before this can be merged to main

**Pull request merged**